etp-backend/src/main/sql/solita/etp/db/laatija.sql

```diff
@@ -8,10 +8,18 @@ SELECT k.id,
        k.henkilotunnus,
        l.patevyystaso,
        l.toteamispaivamaara,
+       l.wwwosoite,
        l.toimintaalue,
+       l.muut_toimintaalueet muuttoimintaalueet,
```

Comment:
nämä saisi mennä kebab casenä

Reply:
Nämäkin on ollut schemassa aina näin, eli jos muutetaan kebab-caseiksi niin oma PR siitä. Silloinhan varmaan pitää tehdä fronttimuutoksia samalla.